{{cookiecutter.project_shortname}}/invenio.cfg

```diff
@@ -0,0 +1,15 @@
+ACCOUNTS_SESSION_REDIS_URL="redis://localhost:6379/1"
+BROKER_URL="amqp://guest:guest@localhost:5672/"
+CACHE_REDIS_URL="redis://localhost:6379/0"
+CACHE_TYPE="redis"
+CELERY_BROKER_URL="amqp://guest:guest@localhost:5672/"
+CELERY_RESULT_BACKEND="redis://localhost:6379/2"
+SEARCH_ELASTIC_HOSTS=[{"host": "localhost", "port": 9200}]
+SECRET_KEY="CHANGE_ME"
+{%- if cookiecutter.database == 'postgresql'%}
+SQLALCHEMY_DATABASE_URI="postgresql+psycopg2://{{cookiecutter.project_shortname}}:{{cookiecutter.project_shortname}}@localhost/{{cookiecutter.project_shortname}}"
+{%- elif cookiecutter.database == 'mysql'%}
+SQLALCHEMY_DATABASE_URI="mysql+pymysql://{{cookiecutter.project_shortname}}:{{cookiecutter.project_shortname}}@localhost/{{cookiecutter.project_shortname}}"
+{%- endif %}
```

Comment:
Shelved: What about SQlite? I made an issue for it: https://github.com/inveniosoftware/cookiecutter-invenio-rdm/issues/16